exercises/simple-cipher/canonical-data.json

```
@@ -1,12 +1,13 @@
 {
   "exercise": "simple-cipher",
-  "version": "1.2.0",
-  "comments":
-    ["Some of the strings used in this file are symbolic and do not represent their literal value. They are:",
-      "cipher.key - Represents the Cipher key",
-      "cipher.encode - Represents the output of the Cipher encode method",
-      "new - Represents the Cipher initialization",
-      "string.substring(start, length) - Represents a substring of 'string' that begins at index 'start' and is 'length' characters long"],
+  "version": "1.2.1",
```

Please review the [test data versioning](https://github.com/exercism/problem-specifications#test-data-versioning) guidelines.

```suggestion
  "version": "1.3.0",
```

---

Thanks. I did read the data versioning guidelines, but it wasn't entirely clear if this was adding new functionality or simply correcting an omission and thus if it would be a minor or patch bump. I've updated to 1.3.0.